dev.zed.Zed.yaml

```diff
@@ -0,0 +1,135 @@
+# dev.zed.Zed.yaml
+app-id: dev.zed.Zed
+runtime: org.freedesktop.Sdk
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: zed
+separate-locales: false
+finish-args:
+  # Grant access to Direct Rendering Infrastructure (DRI) for GPU acceleration
+  - --device=dri
+
+  # Allow access to the user's home directory for reading and writing files
+  - --filesystem=home
+
+  # Allow shared memory access for inter-process communication
+  - --share=ipc
+
+  # Enable network access
+  - --share=network
+
+  # Provide access to X11 display server as a fallback
+  - --socket=fallback-x11
+
+  # Enable access to PulseAudio sound server
+  - --socket=pulseaudio
+
+  # Enable access to Wayland display server
+  - --socket=wayland
+
+  # Enable access to Flatpak host commands (Needs linter exception: https://docs.flathub.org/docs/for-app-authors/linter#exceptions)
+  - --talk-name=org.freedesktop.Flatpak
+
+  # Enable access to Freedesktop Secret Service & related auth services since Zed uses Git heavily
+  - --talk-name=org.freedesktop.secrets
+  - --talk-name=org.kde.kwalletd6
+  - --socket=ssh-auth
+  - --socket=gpg-agent
+  - --talk-name=org.gnome.keyring.SystemPrompter
+
+modules:
+  - name: zed
```

Comment:
`zed` [is now the last module of the manifest, and comments have been reduced.](https://github.com/flathub/flathub/pull/5253/commits/b04373a1ad79fb00cba975020adf1e0feb8213d8)

For future reference, is this because we always want modules to be alphabetical, or is the order supposed to be:

- shared modules
- [other dependencies]
- [the intended software we are packaging]

Reply:
Modules are built in order found. So the main module is always last, with the dependencies in order. Even if that doesn't matter because you decided to not build from source.